pkg/config/derived-service-cert.go

```diff
@@ -20,20 +20,31 @@ import (
 	"strings"
 
 	extutil "github.com/AthenZ/k8s-athenz-sia/v3/pkg/util"
+	"github.com/AthenZ/k8s-athenz-sia/v3/third_party/log"
 )
 
+type X509File struct {
+	Paths []string
```

Comment:
Can we have some kind of comments here
```suggestion
	Paths []string // Stored paths are never empty with splitAndFilterPaths() => Expect paths to be a valid path string.
```

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/192/commits/3aa34307ea4042f642a86ff2c43f9a11620a0398